3303Elevator/src/app/Logger.java

```diff
@@ -115,6 +115,19 @@ public void logTimeManagementSystemEvent(String timeManagementLogString) {
 			}
 		}
 	}
+	/**
+	 * logTimeMeasurements is used to print the Time Measured for a particular process
+	 * 
+	 * @param  string to be printed from the subsystem that has measured the time of a process 
+	 */
+	public void logTimeMeasurements(String time) {
+		String m = getDateTime() + "\n******" + time + "******\n";
```

Comment:
Put newlines on both sides of the stars so it gets printed out like this

**********************
The input string here 
**********************

This will need to be updated in the tests aswell

Reply:
done